README.md

```diff
@@ -114,15 +167,18 @@ BENCHMARK(BM_StringCopy);
 BENCHMARK_MAIN();
 ```
 
-Don't forget to inform your linker to add benchmark library e.g. through 
-`-lbenchmark` compilation flag. Alternatively, you may leave out the 
-`BENCHMARK_MAIN();` at the end of the source file and link against 
-`-lbenchmark_main` to get the same default behavior.
+To run the benchmark, compile and link against the `libbenchmark` library
```

Comment:
This is very tautological.
It's `benchmark` library (`libbenchmark.a/.so`). `libbenchmark` library would be `liblibbenchmark.a/.so`

Reply:
Thanks. Will update.